.azure-pipelines/testscripts_analyse/analyse_testscripts.py

```diff
@@ -183,8 +194,9 @@ def upload_results(test_scripts):
 
 def main():
     test_scripts = collect_all_scripts()
-    topology_type_pr_test_scripts = get_PRChecker_scripts()
-    expanded_test_scripts = check_PRChecker_coverd(test_scripts, topology_type_pr_test_scripts)
+    topology_type_pr_test_scripts, topology_type_pr_test_skip_scripts = get_PRChecker_scripts()
```

Comment:
Nit: `get_PRChecker_scripts` can be renamed to `get_included_skipped_ts_by_topology` that way it also confirms to PEP 8.

Reply:
Do you think `get_included_skipped_ts_by_topology_type` is much better?

Reply:
I think that is indeed very long. I am happy if you can make it shorter or choose another name altogether. I was only looking at removing the acronym caps from the name. `get_PRChecker_scripts` -> `get_pr_checker_scripts` is good too.

Reply:
Let's use `get_pr_checker_scripts`